packages/transactional-adapters/transactional-adapter-pg-promise/src/lib/transactional-adapter-pg-promise.ts

```diff
@@ -27,16 +27,14 @@ export class TransactionalAdapterPgPromise
             fn: (...args: any[]) => Promise<any>,
             setClient: (client?: PgPromiseDbOrTx) => void,
         ) => {
-            if (options) {
-                return pgPromiseDbOrTxInstance.tx(options, (tx) => {
-                    setClient(tx);
-                    return fn();
```

Comment:
By the way, you could also use `options ?? {}` to supply an empty object if no options is provided. This is arguably more readable than using a different txCb based on the presence of options, but I don't have a strong opinion here.

Reply:
I should've figure that out myself. Thanks!